libraries/provision/ansible/playbooks/tasks/install-sg.yml

```diff
@@ -9,4 +9,10 @@
   shell: dpkg -i /tmp/{{ couchbase_sync_gateway_package }}
   when: ansible_distribution == "Ubuntu" or ansible_distribution == "Debian"
 
+- name: SYNC GATEWAY | Install sync_gateway tar.gz
+  become: yes
+  shell: |
+    tar -zxvf /tmp/{{ couchbase_sync_gateway_package }} --directory /opt
```

Comment:
sorry, not a comment, but trying to learn and understand ...... what is the command before the pipe?

Reply:
A command with pipe can be used if you have multiple shell commands. It combines all shell commands and run one after the other.